File: php/URI/3342.php
<?php

$tab = trim(fgets(STDIN));

if ($tab % 2 == 0) {
    $tab = $tab * $tab;
    echo ($tab / 2 . " casas brancas e " . $tab / 2 . " casas pretas\n");
} else {
    $tab = $tab * $tab;
    $tab = $tab / 2;
    echo ((int)$tab + 1 . " casas brancas e " . (int)$tab . " casas pretas\n");
}

File: C/URI/3342.c
#include <stdio.h>

int main()
{
    int tab;

    scanf("%d", &tab);

    if (tab % 2 == 0)
    {
        tab = tab * tab;
        printf("%d casas brancas e %d casas pretas", tab / 2, tab / 2);
    }
    else
    {
        tab = tab * tab;
        tab = tab / 2;
        printf("%d casas brancas e %d casas pretas", tab + 1, tab);
    }
}